DOC:rainfall indices (#369)
DassHydro/smash: doc/source/user_guide/classical_uses/rainfall_indices.rst

Rainfall Indices
================

This section aims to go into detail on how to compute and visualize precipitation indices.

First, open a Python interface:

.. code-block:: none

    python3
    
Imports
-------

> 

Reviewer: Je crois t'as écrit ça pour les autres tutos, c'est bien de le remettre ici je pense.
```suggestion

We will first import everything we need in this tutorial.

```

.. ipython:: python
    
    import smash
    import numpy as np
    import matplotlib.pyplot as plt
    
Model object creation
---------------------

> To compute precipitation indices, you need to create a :class:`smash.Model` object. 
> For this case, we will use the ``Cance`` dataset presented in the User Guide section: :ref:`user_guide.demo_data.cance`.

Reviewer: C'est juste du style pour la cohérence avec les autres tutos.
```suggestion
Now, we need to create a :class:`smash.Model` object.
For this case, we will use the :ref:`user_guide.demo_data.cance` dataset as an example.
```
